### PR TITLE
Add RESTer

### PR DESCRIPTION
**Pull request by wisesascha**
RESTer is a simples easy to use REST client that allows you to test what ever API you want.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Some good apps written with Electron.
 - [Visual Studio Code](https://code.visualstudio.com) - Free cross-platform IDE from Microsoft.
 - [Nylas](https://www.nylas.com/blog/splitting-the-atom) - Email client.
 - [Prepros](https://prepros.io) - Compile almost any preprocessing language with live browser refresh.
-
+- [RESTer](http://getrester.com) - Test REST APIs with an easy to use beautiful tool.
```
Comment (sindresorhus) [inline readme.md]: I prefer the official description:

> Easy and beautiful way to test your APIs and servers.

**Comment by sindresorhus** [comment]
Can you show some proof that it's made with Electron?

**Comment by wisesascha** [comment]
I wrote this article on how I got it into the mac app store.
http://saschawise.com/blog/2015/08/12/electron-for-the-mac-app-store.html .
Heres a screen shot of it with the web inspector http://d.pr/i/1akkS

**Comment by sindresorhus** [comment]
Oh yeah. Hah. I read that article.

File: readme.md (hunk 1 above)
Comment (Glavin001) [inline readme.md]: +1 I also prefer the official description.

Comment (sindresorhus) [inline readme.md]: @wisesascha

**Comment by sindresorhus** [comment]
Can you also fix the merge conflict?

**Comment by sindresorhus** [comment]
@wisesascha You fixed the merge conflict incorrectly. The diff clearly shows this: https://github.com/sindresorhus/awesome-electron/pull/63/files

I've manually landed it.

**Pull request closed without merging**